module-services/service-db/messages/DBServiceMessage.cpp

```diff
@@ -7,5 +7,9 @@ DBServiceMessageBackup::DBServiceMessageBackup(MessageType messageType, std::str
     : DBMessage(messageType), backupPath(backupPath)
 {}
 
+DBServiceMessageSyncPackage::DBServiceMessageSyncPackage(MessageType messageType, std::string syncPackagePath)
```

Comment:
if the path won't be modified consider passing const-ref ;)

Reply:
done